marketplacetests/in_app_payments/in_app.py

```diff
@@ -2,29 +2,67 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette.by import By
 from gaiatest.apps.base import Base
+from gaiatest.apps.base import PageRegion
+from marionette import expected, By, Wait
 
+from marketplacetests.firefox_accounts.app import FirefoxAccounts
 
-class InAppPayment(Base):
+
+class InAppPaymentTester(Base):
+
+    default_server = 'API: marketplace-dev.allizom.org'
 
     # Products
-    _buy_product_button_locator = (By.CSS_SELECTOR, '.item > button')
+    _available_products_locator = (By.ID, 'items')
```

Comment:
It turns out with your other suggestion that we don't even need this anymore. I will remove it.